### PR TITLE
K8SPXC-622 add contributing guide

### PR DESCRIPTION
**Pull request by fiowro**
[K8SPXC-622](https://jira.percona.com/browse/K8SPXC-622) [&#10088;?&#10089;](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| big-data | passed |
| scaling | passed |
| recreate | passed |
| haproxy | passed |
| upgrade-haproxy | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| security-context | passed |
| upgrade-proxysql | passed |
| init-deploy | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| smart-update | passed |
| limits | passed |
| monitoring-2-0 | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| pitr | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | failed |
| demand-backup-encrypted-with-tls | passed |
| upgrade-consistency | passed |
| users | passed |
| tls-issue-self | passed |
| scheduled-backup | failed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |

**Review by spron-in — DISMISSED** [review]

**Review by delgod — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+Before you can contribute, we kindly ask you to sign our [Contributor License Agreement](https://cla-assistant.percona.com/percona/percona-xtradb-cluster-operator) (CLA). You can do this using your GitHub account and one click.
+
+### 2. Code of Conduct
+
+Please make sure to read and observe the Contribution Policy <code-of-conduct.md>.
+
+## Submitting a pull request
+
+Improvement and bugfix tasks for the Operator are tracked in [Jira](https://Jira.percona.com/projects/K8SPXC/issues), Although not mandatory, it is a good practice to examine already open Jira issues before submitting a pull request. For bigger contributions, we suggest creating a Jira issue first and discussing it with the engineering team and community before proposing any code changes. Also, don't forget, that another best place to discuss Percona Kubernetes Operator for Percona XtraDB Cluster with developers and other community members is the [community forum](https://forums.percona.com/categories/kubernetes-operator-percona-xtradb-cluster).
+
```
Comment (spron-in) [inline CONTRIBUTING.md:20]:
```suggestion
Improvement and bugfix tasks for the Operator are tracked in [Jira](https://Jira.percona.com/projects/K8SPXC/issues), Although not mandatory, it is a good practice to examine already open Jira issues before submitting a pull request. For bigger contributions, we suggest creating a Jira issue first and discussing it with the engineering team and community before proposing any code changes. 

Another good place to discuss Percona Kubernetes Operator for Percona XtraDB Cluster with developers and other community members is the [community forum](https://forums.percona.com/categories/kubernetes-operator-percona-xtradb-cluster). 
```

**Review by spron-in — CHANGES_REQUESTED** [review]

**Review by spron-in — APPROVED** [review]

**Review by delgod — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| haproxy | passed |
| upgrade-haproxy | passed |
| big-data | passed |
| recreate | passed |
| scaling | passed |
| scaling-proxysql | passed |
| self-healing | passed |
| security-context | passed |
| init-deploy | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | passed |
| upgrade-proxysql | passed |
| limits | passed |
| monitoring-2-0 | passed |
| demand-backup | passed |
| smart-update | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | passed |
| demand-backup-encrypted-with-tls | failed |
| upgrade-consistency | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |
| scheduled-backup | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| storage | passed |
| upgrade-haproxy | passed |
| big-data | passed |
| haproxy | passed |
| recreate | passed |
| scaling | passed |
| init-deploy | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| security-context | passed |
| restore-to-encrypted-cluster | passed |
| upgrade-proxysql | passed |
| limits | passed |
| monitoring-2-0 | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| operator-self-healing | passed |
| proxysql-sidecar-res-limits | passed |
| pitr | passed |
| users | passed |
| demand-backup-encrypted-with-tls | passed |
| smart-update | failed |
| tls-issue-self | passed |
| tls-issue-cert-manager | passed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | passed |
| scheduled-backup | failed |
| upgrade-consistency | passed |

**Pull request merged**